canonical/canonical.com
# Fix multiple url params not persisting when linked to

**I ran prettier and it formatted a whole bunch of files, I have only worked on `careers-filter-and-sort.js`. But it is worth looking over the other pages to ensure nothing is broken**

## Done

- Fixes bug where if there are more than one url params, only one will be retained

## QA

- Check that this link works https://canonical-com-1155.demos.haus/careers/all?filter=Web+%26+Design&location=americas (retains job and location)
- Try with some other links

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7987
https://warthogs.atlassian.net/browse/WD-8007
